test/e2e-api/http.go

```diff
@@ -42,7 +42,20 @@ func NewHttpController() *HttpController {
 	}
 }
 
+func NewHttpController() *HttpController {
```

Comment:
Do we need to make change to this file? Initialization with an HTTP client should be okay, right?

Reply:
you are right, we do not need to change this file. i will reverse the changes